validator/client/src/test/java/tech/pegasys/teku/validator/client/SyncCommitteeDutyLoaderTest.java

```diff
@@ -109,6 +114,58 @@ void shouldRetrieveDuties() {
                     validator1Index, IntSet.of(1, 6, 25), untilEpoch.increment()),
                 new SyncCommitteeSubnetSubscription(
                     validator2Index, IntSet.of(7, 50, 38), untilEpoch.increment())));
+    assertThat(
+            metricsSystem
+                .getGauge(TekuMetricCategory.VALIDATOR, "scheduled_sync_committee_duties_current")
+                .getValue())
+        .isEqualTo(2.0);
+
+    assertThat(
+            metricsSystem
+                .getGauge(TekuMetricCategory.VALIDATOR, "current_sync_committee_last_epoch")
+                .getValue())
+        .isEqualTo(63.0);
+  }
+
+  @Test
+  void shouldGetCountOfValidatorsInSyncCommitteeThroughMetrics() {
+    final UInt64 epoch = UInt64.valueOf(56);
+    when(validatorApiChannel.getSyncCommitteeDuties(epoch, validatorIndices))
+        .thenReturn(
+            SafeFuture.completedFuture(Optional.of(new SyncCommitteeDuties(false, List.of()))));
+    final SyncCommitteeScheduledDuties duties = loadDuties(epoch);
+    assertThat(duties.countDuties()).isEqualTo(0);
+    assertThat(
+            metricsSystem
+                .getGauge(TekuMetricCategory.VALIDATOR, "scheduled_sync_committee_duties_current")
+                .getValue())
+        .isEqualTo(0.0);
+
+    assertThat(
+            metricsSystem
+                .getGauge(TekuMetricCategory.VALIDATOR, "current_sync_committee_last_epoch")
+                .getValue())
+        .isEqualTo(63.0);
+  }
+
+  @Test
+  void shouldGetCountOfValidatorsInSyncCommitteeThroughMetrics2() {
```

Comment:
Is this method a duplicate of `shouldGetCountOfValidatorsInSyncCommitteeThroughMetrics`?

Reply:
oh yep i remember this, i need to rename it heh